### PR TITLE
change display splash to senseBox logo

### PR DESCRIPTION
**Pull request by mariopesch**
copy senseBox splash to the display library

### CODE REVIEW

**Comment by felixerdy** [comment]
Please add it to the `Dockerfile.test` as well, just to also have it in the test environment

File: splash.h [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,0b11111110,0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,0b01111100,0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,0b00110000,0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,0b00110000,0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,
+};
+
+#define splash2_width  115
+#define splash2_height 32
+
+const uint8_t PROGMEM splash2_data[] = {
```
Comment (mpfeil) [inline splash.h:86]: Is this still the Adafruit Logo from the original library?

If so please replace it with a senseBox logo.

**Review by mpfeil — CHANGES_REQUESTED** [review]

File: splash.h (hunk 1 above)
Comment (mariopesch) [reply splash.h:86]: splash2_data is still the adafruit logo

Comment (mpfeil) [reply splash.h:86]: Then please add a senseBox logo for `splash2_data` too.

**Pull request merged**